packages/node_modules/@webex/internal-plugin-team/src/team.js

```diff
@@ -95,14 +96,16 @@ const Team = WebexPlugin.extend({
    * @param {boolean} options.includeAllTeamMembers
    * @returns {Promise} Resolves with the newly created conversation
    */
-  createConversation(team, params, options) {
-    options = options || {};
+  createConversation(team, params, options = {}) {
+    if (!team.url) {
+      throw Error('`team.url` is required');
+    }
 
     if (!params.displayName) {
       return Promise.reject(new Error('`params.displayName` is required'));
     }
 
-    return this._inferTeamUrl(team)
+    return Promise.resolve()
```

Comment:
Don't need the Promise here, could just return `this._ensureGeneralConversation`